lib/pbench/test/unit/server/database/test_audit_db.py

```diff
@@ -139,10 +143,7 @@ def test_override(self, fake_db):
 
     def test_exceptional_query(self, fake_db):
         FakeSession.throw_query = True
-        with pytest.raises(
-            AuditSqlError,
-            match=r"Error finding {'user_name': 'imme'}: \('test', 'because'\)",
-        ):
+        with pytest.raises(AuditSqlError):
```

Comment:
You're OK with reducing this constraint?

Reply:
Yeah, really, I was, until you went and complained. 😦

Reply:
I wasn't complaining...I was _confirming_....  (I just report what I find...I try to leave the value judgements up to the author when I can.)